forge-gui/res/cardsfolder/upcoming/an_unearthly_child.txt

```diff
@@ -0,0 +1,7 @@
+Name:An Unearthly Child
+ManaCost:1 U U
+Types:Enchantment Saga
+K:Saga:3:DBDigUntil,DBDigUntil,DBDigUntil
+SVar:DBDigUntil:DB$ DigUntil | Valid$ Vehicle,Doctor,Creature.withDoctor's companion  | ValidDescription$ Doctor card, a card with doctor's companion, or a Vehicle card | FoundDestination$ Hand | RevealedDestination$ Library | RevealedLibraryPosition$ -1 | RevealRandomOrder$ True | SpellDescription$ Reveal cards from the top of your library until you reveal a Doctor card, a card with doctor's companion, or a Vehicle card. Put that card into your hand and the rest on the bottom of your library in a random order.
```

Comment:
`Card.with` not Creature.with

Does this ValidDescription even get used as part of a Saga?

Why RevealRandomOrder? I think you want RestRandomOrder

Reply:
@Northmoc 
no, the param name is correct for some reason